### PR TITLE
CO - Changed CDTS version to v5.0.0

### PR DESCRIPTION
**Pull request by dmiraflorhc**

### CODE REVIEW

File: projects/hpfb/sdk/src/assets/static/appFooter-en.html
Comment (linghanHC) [inline projects/hpfb/sdk/src/assets/static/appFooter-en.html:1]: @dmiraflorhc , for files in the assets/static folder, did you copy files from https://cenw-wscoe.github.io/sgdc-cdts/GCWeb/static/Gcwebstatic-v5_0_0.zip?

Comment (dmiraflorhc) [reply projects/hpfb/sdk/src/assets/static/appFooter-en.html:1]: I didn't, I just changed the CDTS/WET versions on the static files in that folder. Would it be better to copy files from the zip instead?

Comment (linghanHC) [reply projects/hpfb/sdk/src/assets/static/appFooter-en.html:1]: yes, please. The files' content could be changed between versions. These files are used when browser's javascript is disabled.

Comment (dmiraflorhc) [reply projects/hpfb/sdk/src/assets/static/appFooter-en.html:1]: Done! Though, could you go over secMenu.html for en and fr? Should it only contain `<p>The menu is not available when Javascript is disabled.</p>`

Comment (linghanHC) [reply projects/hpfb/sdk/src/assets/static/appFooter-en.html:1]: I think whatever the changes they made should be fine. 
btw, https://jill.hc-sc.gc.ca/jira/browse/ART-1240 was resolved automatically when upgraded to v5.0.0

Comment (dmiraflorhc) [reply projects/hpfb/sdk/src/assets/static/appFooter-en.html:1]: Yes, the button is no longer in the form 
image

**Review by AMckayHC — APPROVED** [review]

**Review by AMckayHC — APPROVED** [review]

**Review by AMckayHC — COMMENTED** [review]
Diana, can you remove the change to file: [projects/md-co/src/environments/environment.ts] since it doesn't really apply to this change request?

**Comment by dmiraflorhc** [comment]
> Diana, can you remove the change to file: [projects/md-co/src/environments/environment.ts] since it doesn't really apply to this change request?

Done!

**Pull request merged**